### DISCUSSION TITLE
Is there a way to format text in descriptions

### DISCUSSION THREAD

**Discussion by YulienPohl** [hneemann/Digital#661] [Q&A]
Newline and spacings are ignored.

**Comment by hneemann** [comment]
Formatting with spaces or tabs is not possible.
But if the description is shown in a tool tip only, it is possible to use html like so:

``` html
<html>
Example:
<ul>
<li>Item 1</li>
<li>Item 2</li>
</ul>
</html>
```

But this only works in tool tips.
